app/models/manageiq/providers/ibm_power_hmc/inventory/collector/infra_manager.rb

```diff
@@ -52,8 +53,19 @@ def vnics
     @vnics || {}
   end
 
+  def ssps
+    @ssps || {}
+  end
```

Comment:
Not related to this PR, but all of these look like they could be `attr_accessor` and just set `@ssps = {}` in your `initialize` method then you don't need all these methods